subworkflows/local/annotate_snvs.nf

```diff
@@ -86,7 +86,7 @@ workflow ANNOTATE_SNVS {
         BCFTOOLS_VIEW.out.vcf.join(TABIX_BCFTOOLS.out.tbi).collect().set { ch_vcf_scatter_in }
 
         GATK4_SELECTVARIANTS (ch_vcf_scatter_in.combine(split_intervals)).vcf.set { ch_vep_in }
-        ch_versions = ch_versions.mix(GATK4_SELECTVARIANTS.out.versions)
+        ch_versions = ch_versions.mix(GATK4_SELECTVARIANTS.out.versions.first())
```

Comment:
I haven't used first here because all the other processes in this subworkflow are run only once so the corresponding version channel only has one element.